Entities/Structures/Platform/Bridge.as

```diff
@@ -170,7 +170,7 @@ bool canOpen(CBlob@ this, CBlob@ blob)
 {
 	if (this.getTeamNum() != blob.getTeamNum()
 		&& blob.getShape().getConsts().collidable
-		&& (blob.hasTag("player") || blob.hasTag("vehicle")))
+		&& (blob.hasTag("player") || blob.hasTag("dead") || blob.hasTag("vehicle")))
```

Comment:
Cannot check easily right now, but are you 100% sure that the `dead` tag only ever applies to player blobs?

Reply:
I've never seen it used for anything other than players.

Reply:
Dead fishies have it

Reply:
Does anyone know of a sure-fire way to check for dead bodies? I've always just used this check.

Reply:
Since fishies are team 255, you can do this.

```
        if ((this.getTeamNum() != blob.getTeamNum() && blob.getTeamNum() != 255)
		&& blob.getShape().getConsts().collidable
		&& (blob.hasTag("player") || blob.hasTag("dead") || blob.hasTag("vehicle")))
	{
		print(blob.getTeamNum() + " " + blob.getName());
		return true;
	}
	return false;
```

And if [#1251](https://github.com/transhumandesign/kag-base/pull/1251) gets merged, which adds [TeamChecking.as](https://github.com/transhumandesign/kag-base/blob/130db67cdc411575bf722d5e2d5177faaa19fe96/Entities/Common/Scripts/TeamChecking.as), you can do this:

```
#include "TeamChecking.as"

// (...)

        if (isDifferentTeam(blob, this)
		&& blob.getShape().getConsts().collidable
		&& (blob.hasTag("player") || blob.hasTag("dead") || blob.hasTag("vehicle")))
	{
		print(blob.getTeamNum() + " " + blob.getName());
		return true;
	}
	return false;
```

Migrants, Princess and Trader will not have normal collision with Bridge and fall through in the current build. 
[#1283](https://github.com/transhumandesign/kag-base/pull/1283) may change that. But that is less of a problem since standalone Migrants, Princess and Trader won't show up in normal gameplay.

Reply:
Player blobs are now tagged as `"dead player"` when they die, and this tag is used in the check instead of `"dead"`.